api/src/main/java/org/pmiops/workbench/api/ProfileController.java

```diff
@@ -598,7 +598,10 @@ private void checkUserCreationNonce(DbUser user, String nonce) {
       UpdateContactEmailRequest updateContactEmailRequest) {
     String username = updateContactEmailRequest.getUsername().toLowerCase();
     com.google.api.services.directory.model.User googleUser = directoryService.getUser(username);
-    DbUser user = userDao.findUserByUsername(username);
+    DbUser user =
```

Comment:
nit: Since you do this three times, I'd make a little helper method like `getDbUser()` so these methods are shorter.